nuitka/build/Backend.scons

```diff
@@ -812,6 +812,8 @@ elif env.exe_mode:
     # For NetBSD the rpath is required, on FreeBSD it's warned as unused.
     if isNetBSD():
         env.Append(LINKFLAGS=["-rpath=" + python_lib_path])
+    elif env.relocatable:
```

Comment:
I am thinking, we might just do this all the time, if the goal is to make a specific Python configuration executable. For the later I am talking of accelerated mode. And with the rpath feature being so universally available, there really is no hard need to have the scipt anymore. Already for accelerated mode, some settings are fed into the binary. For standalone that of course is not the case anyway.

Reply:
Yeah just checked, the rpath is not set by accelerated mode scripts right now. And actually setting `PATH` on Windows to be able to load it, was like the only real reason `.CMD` files were ever created there, but the newly added shell scripts are not technically necessary with your change at all, and not even doing what this does.